platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/shared/CheckoutRequest.java

```diff
@@ -42,12 +44,18 @@
     
     void setCreateNew(boolean isCreateNew);
     
-    BranchCheckoutRequest withCreateNew(boolean isCreateNew);
+    CheckoutRequest withCreateNew(boolean isCreateNew);
 
     /** @return name of branch that will be tracked */
     String getTrackBranch();
 
     void setTrackBranch(String trackBranch);
 
-    BranchCheckoutRequest withTrackBranch(String trackBranch);
+    CheckoutRequest withTrackBranch(String trackBranch);
+    
+    /** @return list of files to checkout */
+    List<String> getFiles();
+    
+    void setFiles(List<String> files);
+
```

Comment:
In this class you can see that each field has 3 methods: getter, setter and method `CheckoutRequest withField(FiledType fieldValue)` which implements fluent pattern.
I asked you to add this `CheckoutRequest withFiles(List<String> files)`